src/pages/Notes.jsx

```diff
@@ -4,21 +4,53 @@ import { PageNotFound } from "./PageNotFound";
 import "../stylesheets/notes.css";
 import { AddNote } from "../components/AddNote";
 import { useMessageHandling } from "../context/message-handling-context";
+import { useState } from "react";
+import { useDBdata } from "../context/db-data-context";
 
 export function Notes() {
   const location = useLocation();
   const background = location.state && location.state.background;
   const { showErrorPage } = useMessageHandling();
-
+  const { notes } = useDBdata();
+  const [showSortBy, setShowSortBy] = useState(false);
+  const [sortBy, setSortBy] = useState("Latest");
   return (
     <div>
       <Navbar />
+      {notes?.length > 0 && (
+        <header className="header">
```

Comment:
Can be imported as a separate component for ease of reading

Reply:
Done